tripal/tripal
# Bug Fixes pre v3.5 release.

# Bug Fix 
This PR adds a documentation fix and a bug fix to the GFF3 loader that I noticed when I was testing Tripal prior to the v3.5 release.

The problem is that the GFF3 loader was not adding all of the records to the `feature_relationship` table.   The issue is caused by the fact that the GFF3 loader organizes children of each parent by their start location, but if two children have the same start location it ends up only keeping track of one of them.  This PR fixes the problem by storing the children in an array.  I'm amazed we missed this with all the testing we did!

# Testing
1 Start with a brand new Tripal site or a site that does not have the demo citrus data loaded.
2. Import the Citrus demo data that comes with the User's Guide
3. Publish the genes
4. Go to the gene page and click on the transcript and the relationships links. You'll see only one mRNA.
![image](https://user-images.githubusercontent.com/1719352/106079640-7525bd80-60ca-11eb-9b56-e5ee0d59ce57.png)
5. Pull this PR, rerun the GFF3 import
6. Go back to the gene page, click the reload tab, and now all the transcripts are present.
![image](https://user-images.githubusercontent.com/1719352/106079739-a1d9d500-60ca-11eb-9b6e-ce2a49828161.png)
